src/components/details/Cast.jsx

```
@@ -1,8 +1,73 @@
-/* eslint-disable react/prop-types */
-import React from 'react';
+import React, { useRef } from 'react';
 
 export default function Cast({ details }) {
```

ESLint

[error] 'details' is missing in props validation

(react/prop-types)